### PR TITLE
#167 - Updated Podfile

### PR DESCRIPTION
**Pull request by mattbarker016**
#167 Updated the following dependencies to their new @cuappdev forks.

- [x] Fuzzywuzzy_swift
- [x] Presentation
- [x] NotificationBanner
- [x] DZNEmptyDataSet

@ywsang Once your transfer the forked repo, you just need to clone this branch and change "ywsang" to "cuappdev" in the `:git` entry in the Podfile. You can do this by going to your [fork for DZNEmptyDataSet](https://github.com/ywsang/DZNEmptyDataSet), clicking on Settings, scrolling down and Transfer Ownership of the repo to `cuappdev`.

### CODE REVIEW

**Review by AAAstorga — APPROVED** [review]
for future reference, if you are just modifying project settings, no need to get my approval. I trust your judgement

**Comment by mattbarker016** [comment]
This is ready to merge, will do so after #150

**Pull request merged**
